src/item.cpp

```diff
@@ -2493,16 +2493,11 @@ int item::bash_resist() const
         return resist;
     }
     if (item::item_tags.count("leather_padded") > 0){
-        l_padding = volume() * (float(get_coverage()) / 100) / 2.5;
-        if (l_padding > 5 ){
-            l_padding = l_padding / 2.5;   //Hard cap so coats don't become solid steel
-        }
+               //Max value           Steepness of graph         Center of 'S'
```

Comment:
If it's logically the same value, it should only be defined once. That means if the same logic with the same values applies to both leather and Kevlar (which btw. seems strange, shouldn't Kevlar be better?), define the values at the beginning of the functions.

If Kevlar and leather should use different values (they are just accidentally the same as of now), define them right before the are used as they are not required outside of the block.

You could also declare them at a global scope inside item.cpp so they can be used in both functions.